networked_controller.cpp

```diff
@@ -1191,7 +1259,7 @@ int AutonomousServerController::get_inputs_count() const {
 bool AutonomousServerController::fetch_next_input(real_t p_delta) {
 	node->get_inputs_buffer_mut().begin_write(METADATA_SIZE);
 	node->get_inputs_buffer_mut().seek(METADATA_SIZE);
-	node->call(SNAME("_collect_inputs"), p_delta, &node->get_inputs_buffer_mut());
+	node->native_collect_inputs(p_delta, node->get_inputs_buffer_mut());
```

Comment:
just to ensure: will it still collect inputs of gdscript implementation here or it's not needed?

Reply:
(apply also to other changes around)

Reply:
I assume it will do, since there is `GDVIRTUAL_CALL(_collect_inputs, p_delta, &r_buffer);` inside `native_collect_inputs()` so I think it's the new way of calling gdscript from c++?

Reply:
`native_collect_inputs` is just a wrap function that can be overridden from C++. If not, the GDS function is normally called using `GDVIRTUAL_CALL(_collect_inputs, p_delta, &r_buffer); inside native_collect_inputs()`